.gitignore

```diff
@@ -1,13 +1,11 @@
-_cache/
-brigade-api/rootfs/brigade-api
-brigade-controller/rootfs/brigade-controller
-brigade-github-gateway/rootfs/brigade-github-gateway
-brigade-cr-gateway/rootfs/brigade-cr-gateway
-brigade-vacuum/rootfs/brigade-vacuum
-brigade-worker/dist
-brigade-worker/doc/
-bin/
+/bin/
+/brigade-api/rootfs/brigade-api
+/brigade-controller/rootfs/brigade-controller
+/brigade-cr-gateway/rootfs/brigade-cr-gateway
+/brigade-github-gateway/rootfs/brigade-github-gateway
+/brigade-vacuum/rootfs/brigade-vacuum
+/brigade-worker/dist/
+/brigade-worker/doc/
+/tests/testdata/*-generated.*
+/vendor/
 node_modules/
```

Comment:
Nit: Perhaps you'd also want to chage this like `/node_modules/`?

Reply:
I think that without the slash, it matches any `node_modules/` directory, while with the slash it just matches a directory at the root.

Reply:
yep, without the `/` it is recursive.

Reply:
I was too blind! Totally understood. Thank you very much for the explanation.